testing/kuttl/e2e/pgbouncer/00-assert.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: pgbouncer-test
+status:
+  instances:
+    - name: instance1
+      readyReplicas: 2
+      replicas: 2
+      updatedReplicas: 2
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    postgres-operator.crunchydata.com/cluster: pgbouncer-test
+    postgres-operator.crunchydata.com/pgbackrest-backup: replica-create
+status:
+  succeeded: 1
```

Comment:
❓ I didn't expect this job to have any effect on PgBouncer. What does its success mean here?

Reply:
In the other tests we've used this assert to mean that the cluster is up and running, though taking a step back I wonder if this is necessary/the best way to check that status.